test/src/runner/test-runner/regular-test-runner.js

```diff
@@ -122,7 +128,10 @@ describe("runner/test-runner/regular-test-runner", () => {
         });
 
         it("should modify state if 'testXReqId' is not actual", async () => {
-            const state = { testXReqId: "12345" };
+            const state = {
+                testXReqId: "12345",
+                traceparent: "00-11111111111111111111111111111111-0011111111111111-01",
+            };
```

Comment:
Test is called "should modify state if 'testXReqId' is not actual", but there is `traceparent` check also.

I would write separate test for "traceparent", because its unobvious that this test will fall if `traceparent` is broken.